test/libsolidity/semanticTests/unused_store_storage_removal_bug.sol

```diff
@@ -0,0 +1,15 @@
+contract C {
+	uint public x;
+	function f() public {
+		x = 1; // This write used to be removed by the yul optimizer.
+		g();
+		x = 1;
```

Comment:
```suggestion
		x = 2;
```

Reply:
Just so that the removal becomes obvious.

Reply:
Nevermind. The assembly `return(0, 0)` implies `x = 0` so retaining the `1` is fine. I just felt the redundancy could be made more clear by writing a different value but I see that it is not required for understanding the bug.

Reply:
I think it would still be a bit clearer to make it `2` so that it's completely obvious which writes were executed.